app/src/main/java/com/sopt/now/domain/usecase/FetchUserInfoUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.sopt.now.domain.usecase
+
+import com.sopt.now.domain.model.UserInfo
+import com.sopt.now.domain.repository.UserInfoRepository
+import javax.inject.Inject
+
+class FetchUserInfoUseCase @Inject constructor(
+    private val userInfoRepository: UserInfoRepository
+) {
+    suspend fun execute(memberId: String): UserInfo {
+        return userInfoRepository.fetchUserInfo(memberId)
+    }
```

Comment:
```suggestion
    suspend fun execute(memberId: String): UserInfo =
        userInfoRepository.fetchUserInfo(memberId)
```
잘 하셔서 크게 리뷰 남길 게 없다가 뭐라도 써야할 것 같아서 씁니다. 이 내용은 취향 차이라 흘려 들으셔도 무방합니다.